sdk/src/wallet/core/operations/storage.rs

```diff
@@ -84,6 +84,14 @@ mod storage_stub {
             Ok(res.map(Into::into))
         }
     }
+
+    impl Wallet {
+        pub async fn get_chrysalis_data(
```

Comment:
delete

Reply:
Remove locally, no need to further comment about it